additional_recipes/dynmsg/build.sh

```diff
@@ -0,0 +1,141 @@
+# Generated by vinca http://github.com/RoboStack/vinca.
+# DO NOT EDIT!
```

Comment:
If we want to maintain this independently from vinca, probably it make sense to remove this comment?

Reply:
Agreed!